docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -53,6 +53,19 @@ The global profile functionality simplifies profile management by enabling you t
 
 **Note**: A project context takes precedence over global configuration.
 
+### Create a Team Configuration File
+
+Create a team configuration file.
+
+1. Navigate to the explorer tree.
+2. Hover over **DATA SETS**, **USS**, or **JOBS**.
+3. Click the **+** icon.
+4. Select **Create a New Team Configuration File**.
+5. Follow the instructions to complete the team configuration profile creation.
+6. Reload your VS Code window when prompted.
+
```

Comment:
I think a # 7 should be added here to right-click the profile loaded from config file after the reload and select update to update the config file with host and any other fields needed.  Then the user would have to reload again to pick up those changes.

Reply:
this info could be moved to the `### Manage a Profile`  section

Reply:
yeah, this step is indicated in the Manage a Profile section